src/converter-gen/nist-metaschema-MAKE-JSON-TO-XML-CONVERTER.xsl

```diff
@@ -84,175 +48,16 @@
     <nm:transform version="3.0">{ $composer-dir }/reduce-map.xsl</nm:transform>
     
     <nm:transform version="3.0">json-to-xml/produce-json-converter.xsl</nm:transform>
-    <!--<nm:transform version="3.0">package-json-converter.xsl</nm:transform>-->
+    <nm:transform version="3.0">json-to-xml/package-json-converter.xsl</nm:transform>
   </xsl:variable>
   
   <xsl:variable name="metaschema-source" select="/"/>
   
   <xsl:template match="/">
-    <xsl:variable name="json-converter">
-      <xsl:call-template name="nm:process-pipeline">
-        <xsl:with-param name="sequence" select="$produce-json-converter"/>
-      </xsl:call-template>
-    </xsl:variable>
-    <xsl:apply-templates select="$json-converter" mode="package-converter"/>
+    <xsl:comment expand-text="true"> XSLT generated by { replace($xslt-base,'.*/','') }, { current-dateTime() } </xsl:comment>
+    <xsl:call-template name="nm:process-pipeline">
+      <xsl:with-param name="sequence" select="$produce-json-converter"/>
+    </xsl:call-template>
```

Comment:
Great change! I'm studying the converter-gen process, and I wanted to look at the intermediate states by having the XSLT save them as files. Because `xsl:result-document` (from my temporary local modification of the downstream code that calls `transform()`) can't go inside the `xsl:variable` on the old line 93, even indirectly, my task is easier with this change, because you're making the package-converter processing just another pipeline step without the XSLT variable.

Reply:
Yep, better is just better, isn't it.

FWIW, an intervention in the handler XSLT at https://github.com/usnistgov/metaschema-xslt/blob/develop/src/common/nist-metaschema-metaprocess.xsl could provide for the intermediate steps to be written out, not just executed sequentially. Indeed it already supports an 'alert' option that might be extended with `xsl:result-document` ...

Reply:
Yes, that's the file where my temporary local modification is. :)

Reply:
(Shoulda figured you were ahead of me.)